app-utils/pay-respects/autobuild/beyond

```diff
@@ -0,0 +1,7 @@
+abinfo "Installing AI & runtime rules extension ..."
+mkdir -vp "$PKGDIR"/usr/lib/pay-respects
```

Comment:
I think you actually meant `/usr/libexec`:

https://wiki.aosc.io/zh/developer/packaging/package-styling-manual/#wen-jian-fang-zhi